packages/api-nest/src/queue/services/content-type-detector.service.spec.ts

```
@@ -0,0 +1,210 @@
+/**
+ * ContentTypeDetectorService Unit Tests
+ */
+
+import { Test, TestingModule } from '@nestjs/testing'
+import { ContentTypeDetectorService } from './content-type-detector.service'
+import { ContentType } from '../../library/entities/library-item.entity'
```

ESLint

[error] Run autofix to sort these imports!

(simple-import-sort/imports)